bin/node/src/chain_spec.rs

```diff
@@ -244,7 +254,8 @@ fn generate_chain_spec_config(
                 authorities.clone(), // Initial PoA authorities, will receive funds
                 sudo_account.clone(), // Sudo account, will also be pre funded
                 faucet_account.clone(), // Pre-funded faucet account
-                stakers.clone(),
+                controller_accounts.clone(), // Controller accounts for staking.
```

Comment:
This is what I had thought as well, however, the compiler complains:

  error[E0507]: cannot move out of `controller_accounts`, a captured variable in an `Fn` closure
     --> bin/node/src/chain_spec.rs:257:17
      |
  234 |       controller_accounts: Option<Vec<AccountId>>,
      |       ------------------- captured outer variable
  ...
  251 | /         move || {
  252 | |             generate_genesis_config(
  253 | |                 wasm_binary,
  254 | |                 authorities.clone(), // Initial PoA authorities, will receive funds
  ...   |
  257 | |                 controller_accounts, // Controller accounts for staking.
      | |                 ^^^^^^^^^^^^^^^^^^^ move occurs because `controller_accounts` has type `std::option::Option<Vec<AccountId32>>`, which does not implement the `Copy` trait
  258 | |                 stash_accounts, // Stash accounts for staking.
  259 | |             )
  260 | |         },
      | |_________- captured by this `Fn` closure
      |
  help: consider borrowing the `Option`'s content
      |
  257 |                 controller_accounts.as_ref(), // Controller accounts for staking.
      |                                    +++++++++

Same for the latter one.